lab_1_classify_by_unigrams/main.py

```diff
@@ -64,12 +92,22 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
-
-
-def compare_profiles(
-    unknown_profile: dict[str, str | dict[str, float]],
-    profile_to_compare: dict[str, str | dict[str, float]],
-) -> float | None:
+    if not isinstance(predicted, list) or not isinstance(actual, list):
+        return None
+    if len(actual) != len(predicted):
+        return None
+    summa = 0.0
+    for index, value in enumerate(actual):
+        actual_value = float(value)
```

Comment:
Does this really need to be explicitly converted to float?

Reply:
done